new/LIT7305LaEmmaBenyam.xml

```diff
@@ -0,0 +1,83 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7305LaEmmaBenyam" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ለእመ፡ ብንያም፡ ወዮሴፍ፡ ዘአዕይንቲሃ፡ ኮከብ፡ በመዝአረ፡ ገጻ፡ ዘአሶንያ፡</title>
+                <title xml:lang="gez" type="normalized" corresp="#t1">La-ʾǝmma Bǝnyām wa-Yosef za-ʾaʿǝyyǝntihu kokab ba-mazʾaru gaṣā za-ʾAsonyā (Qǝne of ʿǝṭāna mogar gǝʿǝz-type)</title>
+                
+                <title xml:lang="en" corresp="#t1">For the mother og Benyam and Joseph, whose eyes are stars...</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="DN"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p>The type of this qene poem is indicated as <foreign xml:lang="gez">ʿǝṭāna mogar</foreign> 
+                    performed in the melodic mode <term key="GeezMode">gǝʿǝz</term>.</p>
```

Comment:
one could add here or in the relations the pointers to Rachel, Benjamin, Joseph

Reply:
ok